wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary">
+        <v-card-text> Settings </v-card-text>
+      </v-sheet>
+      <v-card-text>
+        Dark/Light:   
+        <v-btn 
+          icon="mdi-theme-light-dark" @click="toggleTheme" />
+      </v-card-text>
+      
+      <v-sheet class="ml-6">
+        <v-select
+          color="secondary"
+          v-model="chosenTheme"
+          label="Choose a Theme"
+          :items="['Standard', 'Mystic Meadow', 'Enchanted Eclipse']"
+        ></v-select>
+        <v-btn color="secondary" @click="changeTheme" class="mb-6" type="submit">Activate</v-btn>
+      </v-sheet>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { useTheme } from "vuetify";
+import nuxtStorage from "nuxt-storage";
+
+const theme = useTheme();
+const modelValue = defineModel<boolean>({ default: false });
+const chosenTheme = ref('Standard');
+
+
+function changeTheme() {
+  const darkMode: boolean = isDark();
+  if(chosenTheme.value === "Mystic Meadow"){
+    if(darkMode){
+      theme.global.name.value = "mysticMeadowDark";
+    }
+    else {
+      theme.global.name.value = "mysticMeadowLight";
+    }
+  }
+  if(chosenTheme.value === "Enchanted Eclipse"){
+    if(darkMode){
+      theme.global.name.value = "enchantedEclipseDark";
+    }
+    else {
+      theme.global.name.value = "enchantedEclipseLight";
+    }
+  }
+  if(chosenTheme.value === "Standard"){
+    if(darkMode){
+      theme.global.name.value = "dark";
+    }
+    else {
+      theme.global.name.value = "light"
+    }
+  }
+  nuxtStorage.localStorage.setData("theme", theme.global.name.value);
+}
+
+function isDark(): boolean {
+  const last4Letters = theme.global.name.value.substring(theme.global.name.value.length - 4).toLowerCase();
+  return last4Letters === "dark";
```

Comment:
```suggestion
  return new RegExp("dark$|^dark$", "i").test(theme.global.name.value);
```

Can use regex instead. This checks if the theme is a dark variant or if it is the standard dark theme. It is also case insensitive so there is no need to do call `toLowerCase()`

Reply:
I'm not super familiar with regex so for now I'm going to keep using the sub-string method, but if I do gain a better understanding of it I will definitely implement it.